docs/about-mina/faq.mdx

```diff
@@ -16,19 +16,15 @@ Answers to frequently asked questions about Mina Protocol.
 
 ### What language is Mina written in?
 
-The Mina node is written in OCaml. You don't need to know OCaml to write smart contracts for Mina.
+The Mina node is written in OCaml and RUST. You don't need to know OCaml/RUST to write smart contracts for Mina.
```

Comment:
Rust shouldn't be spelled all upper case

Reply:
Updated to Rust